packages/rocketchat-videobridge/server/actionLink.js

```diff
@@ -0,0 +1,3 @@
+RocketChat.actionLinks.register('joinJitsiCall', function(message, params) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["params" is defined but never used](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2734190374/issues/source?bid=3199634&fileBranchId=3342079#l1)